docarray/array/storage/elastic/find.py

```diff
@@ -123,3 +131,24 @@ def _find(
             query = query.reshape((num_rows, -1))
 
         return [self._find_similar_vectors(q, limit=limit) for q in query]
+
+    def _find_with_filter(self, query: Dict, limit: int = 10):
+        resp = self._client.search(
+            index=self._config.index_name,
+            query=query,
+            size=limit,
+        )
+
+        list_of_hits = resp['hits']['hits']
+
+        da = DocumentArray()
+        for result in list_of_hits[:limit]:
+            doc = Document.from_base64(result['_source']['blob'])
+            doc.scores['score'] = NamedScore(value=result['_score'])
+            da.append(doc)
+
+        return da
+
+    def _filter(self, query: Dict, limit: int = 10) -> 'DocumentArray':
+
+        return self._find_with_filter(query, limit=limit)
```

Comment:
can't we just put the implementation of _find_with_filter directly inside _filter ?

Reply:
this is just aligned with the interface since we have this `_filter` function in other backends and also in `mixin/find.py`.